HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -91,6 +90,27 @@ string custodianCode
             };
             SendEmail(emailModel);
         }
+        public void SendReferenceCodePendingLAEmail
```

Comment:
Let's call these methods `SendReferenceCodeEmailForPendingLocalAuthority()` and `SendReferenceCodeEmailForLiveLocalAuthority()`